app/models/micropost.rb

```diff
@@ -0,0 +1,19 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+
+  scope :recent_posts, ->{order(created_at: :desc)}
```

Comment:
```suggestion
  scope :recent_posts, ->{order created_at: :desc}
```
ngoài ra, scope viết sau validate cụ ơi

Reply:
ready ạ